fsspec/caching.py

```diff
@@ -766,15 +766,45 @@ def _read_cache(self, start, end, start_block_number, end_block_number):
             return b"".join(out)
 
 
-caches = {
-    "none": BaseCache,
-    None: BaseCache,
-    "mmap": MMapCache,
-    "bytes": BytesCache,
-    "readahead": ReadAheadCache,
-    "block": BlockCache,
-    "first": FirstChunkCache,
-    "all": AllBytes,
-    "parts": KnownPartsOfAFile,
-    "background": BackgroundBlockCache,
-}
+caches = {}
```

Comment:
I think it would be good to keep `None` here rather than have the extra logic in `add_cache` for this case.